conf/sample.ini

```diff
@@ -495,6 +495,16 @@
 ;sync_cron = "0 0 1 * * *"
 ;active_sync_enabled = true
 
+#################################### AWS ###########################
+[aws]
+# Enter a comma-separated list of allowed AWS authentication providers. 
+# Options are: default (AWS SDK Default), keys (Access && key secret key), credentials (Credentials field), ec2_IAM_role (EC2 IAM Role)
```

Comment:
```suggestion
# Options are: default (AWS SDK Default), keys (Access key && secret key), credentials (Credentials field), ec2_IAM_role (EC2 IAM Role)
```

Reply:
It's actually called `Access & secret key` in the data source config dropdown, so we should use that instead

Reply:
OK, adjusting my suggestion.